src_assets/common/assets/web/index.html

```diff
@@ -35,3 +73,43 @@ <h2>Legal</h2>
     </div>
   </div>
 </div>
+
+<script>
+  new Vue({
+    el: "#content",
+    data() {
+      return {
+        version: null,
+        githubVersion: null,
+        nightlyData: null,
+        loading: true,
+      }
+    },
+    async created() {
+      try {
+        this.version = (await fetch("/api/config").then((r) => r.json())).version;
+        this.githubVersion = (await fetch("https://api.github.com/repos/LizardByte/Sunshine/releases/latest").then((r) => r.json()));
+        if (this.version.split("-g").length > 1) {
+          this.nightlyData = (await fetch("https://api.github.com/repos/LizardByte/Sunshine/actions/workflows/CI.yml/runs?branch=nightly&status=success&per_page=1").then((r) => r.json())).workflow_runs[0];
+        }
+      } catch(e){
+      }
+      this.loading = false;
+    },
+    computed: {
+      hasNewStable() {
+        if (!this.githubVersion || !this.version) return false;
+        if (this.version.indexOf("dirty") !== -1) return false;
+        let v = this.githubVersion.name;
+        if (v.indexOf("v") === 0) v = v.substring(1);
+        return v !== this.version.split("-")[0];
+      },
+      hasNewNightly() {
+        if (!this.version || !this.nightlyData || this.version.split("-g").length === 1) return false;
+        if (this.version.indexOf("dirty") !== -1) return false;
+        let commit = this.version.split("-g")[1].split("-")[0];
+        return this.nightlyData.head_sha.indexOf(commit) !== 0;
+      }
+    }
+  });
+</script>
```

Comment:
please add empty blank line at end of file

Reply:
Done